### PR TITLE
422/account design updates

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses [#472](https://github.com/metrotranscom/doorway/issues/472)

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR intended to cover the feedback described in the note with the exception of one change. The horizontal padding on the confirmation page will stay as 2 rem because it uses the FormSummaryDetails component which is used throughout the application flow where the padding is 2rem throughout.

## How Can This Be Tested/Reviewed?

Go through each change numbered in the ticket and then match the PR commented below the ticket.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/src/components/shared/FormSummaryDetails.tsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ const FormSummaryDetails = ({
 
   return (
     <>
-      <h3 className="form--card__sub-header">
+      <h3 className="form--card__sub-header border-none text-xl">
```
Comment (emilyjablonski) [inline sites/public/src/components/shared/FormSummaryDetails.tsx:190]: We can return to this later, but flagging to shift to the Seeds Heading component.

**Review by emilyjablonski — APPROVED** [review]
Should the padding on the create account page match the account settings page? It may need a rebase

File: sites/public/src/pages/account/StatusItemWrapper.tsx [hunk 2]
```diff
@@ -11,11 +11,11 @@ interface StatusItemWrapperProps {
...
 
 const StatusItemWrapper = (props: StatusItemWrapperProps) => {
+  const applicationDueDate = props.application?.fullListing?.applicationDueDate
+
   return (
     <StatusItem
-      applicationDueDate={dayjs(props.application?.fullListing?.applicationDueDate).format(
-        "MMMM D, YYYY"
-      )}
+      applicationDueDate={!applicationDueDate && dayjs(applicationDueDate).format("MMMM D, YYYY")}
```
Comment (ludtkemorgan) [inline sites/public/src/pages/account/StatusItemWrapper.tsx:18]: This is sometimes returning Invalid Date
image

**Review by ludtkemorgan — COMMENTED** [review]
The styles all look good. Just one issue with the due date

File: sites/public/src/pages/account/StatusItemWrapper.tsx (hunk 2 above)
Comment (ColinBuyck) [reply sites/public/src/pages/account/StatusItemWrapper.tsx]: Ah good catch, I had that "!" in there for testing. Should work now 🐬

**Review by ludtkemorgan — APPROVED** [review]

**Pull request merged**